<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project</class>
 <widget class="QWidget" name="Project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>320</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>269</width>
     <height>320</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Compiler</string>
    </attribute>
    <widget class="QLabel" name="labelmem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>41</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Memory Mode</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxMemoryMode">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelOptimization">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>71</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Optimization</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxOptimization">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelcompiler">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Compiler Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCompiler">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMathlib">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Math Lib</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox32bitDouble">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>32bit Double</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPthreadLib">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pthread Lib</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSimplePrintf">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Simple printf</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxWarnAll">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>All Warnings</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxNoFcache">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>No FCache</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxExceptions">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Exceptions</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxStripELF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Strip ELF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCompOptions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Other Compiler Options</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLinkOptions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Other Linker Options</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
